images/html-editor/accessibility-check.svg

```diff
@@ -0,0 +1,3 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```

Comment:
Could you follow the steps in the [contribution guide](https://github.com/BrightspaceUI/icons#updating-or-contributing-new-icons)? Specifically: `id` should be removed and width/height should be added.